tests/kernel_bundle/sycl_build_zero_device.cpp

```diff
@@ -0,0 +1,60 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  This test calls sycl::build(kernel_bundle<>, std::vector<device>,
+//  property_list) with empty device vector.
+//
+//  The test verifies that an exception with sycl::errc::invalid was thrown.
+//
+*******************************************************************************/
+
+#include "../common/assertions.h"
+#include "../common/common.h"
+#include "kernel_bundle.h"
+#include "kernels.h"
+#include "sycl_build.h"
+
+#define TEST_NAME sycl_build_zero_device
+
+namespace TEST_NAMESPACE {
+using namespace sycl_cts;
+using namespace kernels;
+using namespace sycl_cts::tests::sycl_build;
+using namespace sycl_cts::tests::kernel_bundle;
+
+class TEST_NAME : public sycl_cts::util::test_base {
+ public:
+  /** return information about this test
+   */
+  void get_info(test_base::info &out) const override {
+    set_test_info(out, TOSTRING(TEST_NAME), TEST_FILE);
+  }
+
+  /** execute the test
+   */
+  void run(util::logger &log) override {
+    auto q = util::get_cts_object::queue();
+
+    std::vector<sycl::device> zero_device{};
```

Comment:
```suggestion
    std::vector<sycl::device> zero_device;
```

Reply:
I've applied this comment in this commit: https://github.com/KhronosGroup/SYCL-CTS/pull/228/commits/546b2aa975eb879a6330732e1b60ab310626ef7b